### DISCUSSION TITLE
How to customise PDB loading for modified PDB files?

### DISCUSSION THREAD

**Discussion by l-Dr-MR-l** [a-r-j/graphein#395] [Q&A]
I've got some edited PDB files that contain additional Amino Acid types (HSP for example which is a protonated HIS) which hold some extra information. 
I was wondering if there's some easy way to add to the list of recognised AA's so that I can load these with graphein?

**Comment by a-r-j** [comment]
Hi @l-Dr-MR-l, I'm a little occupied at the moment but you could contribute to this PR: https://github.com/a-r-j/graphein/pull/370

You can find a reference to the list of recognised AAs here: https://github.com/a-r-j/graphein/issues/387#issuecomment-2066694498

**Reply by l-Dr-MR-l** [reply]
Thanks for the suggestion @a-r-j, I must admit I'm not so experienced with github.
I've pulled the PR and made a change adding HSP. How do I contribute this to the existing PR?
